### DISCUSSION TITLE
Create per-device polls and link to them from docs?

### DISCUSSION THREAD

**Discussion by deviantintegral** [Koenkk/zigbee2mqtt#20395] [General]
The Devices documentation is an excellent resource. Not only is it great for zigbee2mqtt users, but it's a good resource for anyone using Zigbee devices. The per-device tips ("how do you pair or reset") are also helpful.

However, what's missing is any sort of way to determine how successful users are with each device. For example, I purchased two of these smart knobs: https://www.zigbee2mqtt.io/devices/ERS-10TZBVK-AA.html#tuya-ers-10tzbvk-aa

I found that unfortunately, the physical hardware for the toggle click action is poor. On both dials, even the slightest movement during a click will cause a rotation / brightness change event to be sent instead. If the lights are off, nothing happens.

What I'm imagining is using polls as a very light-weight analytics system. Something like:

- Create a general poll for each device.
- Ask users to vote if they would recommend, if they wouldn't, some other option for "it depends".
- Ask users to comment and link to any issues they've had.
- Link to the polls from the documentation pages.
- In the future, perhaps link to them somehow from the front-end?
- Likewise, in the future we could consume the votes data back into the docs, or even the front-end itself.

Thoughts? I suppose this could start out very generally as a template, and creating some topics for 10-20 devices. See what happens, let the community lead, and then consider bulk-creating topics for all supported devices if there's success.

**Comment by Koenkk** [comment]
Sounds good! But I'm not sure if we can easily set this up for zigbee2mqtt.io since it has no backend (it's just a static website)

**Reply by deviantintegral** [reply]
I think it would have to be something to add during the production builds that calls the GitHub API or CLI. Or, it could just be a step in the PR for adding any new device support. That would also have the side effect of subscribing the device contributor to the discussion by default.

I took a pass at creating a poll at https://github.com/Koenkk/zigbee2mqtt/discussions/20399. The template would be something like:

```
Title: DEVICE NAME
Category: Device Feedback

If you have used a [DEVICE NAME](https://www.zigbee2mqtt.io/devices/...) with zigbee2mqtt, please vote below with your feedback.

If you answer "No" or "It depends", please share your feedback below. Links to any issues, bug reports, or other discussions are helpful.

To change your vote at a later date, click "Hide Results" below the poll and choose a new option.
```

**Reply by Koenkk** [reply]
And then create a discussion for every supported device?

**Reply by deviantintegral** [reply]
That’s honestly the part that gives me the most pause. I don’t like the idea of creating hundreds of threads that could end up never being used.

Here’s an idea: What if I search for the most active GitHub discussions related to devices, and we create some threads for those? If they get good engagement and are valuable, then we could start to proactively create discussion threads.

**Reply by Koenkk** [reply]
>  If they get good engagement and are valuable, then we could start to proactively create discussion threads.

Having to do this manually is not going to work, I think it would be better to look into poll options directly on Zigbee2MQTT.io but then we probably need to embed something (since it has no backend)
